src/test/java/mat/client/shared/MeasureLibraryGridToolbarTest.java

```diff
@@ -109,6 +110,15 @@ public void testExportOnEmptySelection() {
         Mockito.verify(toolbar.getExportButton(), Mockito.times(1)).setWidth(Mockito.eq("72px"));
     }
 
+    public void testApplyDefaultFhirValidationButton() {
```

Comment:
Looks like the test is not annotated. I will not run

Reply:
Added it